doc/source/dev/roadmap-detailed.rst

```diff
@@ -346,24 +345,18 @@ isolve:
 
 dsolve:
 
-- add sparse Cholesky or incomplete Cholesky
-- add sparse QR
+- add license-compatible sparse Cholesky or incomplete Cholesky
+- add license-compatible sparse QR
 - improve interface to SuiteSparse UMFPACK
 - add interfaces to SuiteSparse CHOLMOD and SPQR
 
-Ideas for new features:
-
-- Wrappers for PROPACK for faster sparse SVD computation.
-
 
 spatial
 ```````
-QHull wrappers are in good shape, as is ``cKDTree``.
+QHull wrappers are in good shape, as is ``KDTree``.
 
 Needed:
 
-- ``KDTree`` will be removed, and ``cKDTree`` will be renamed to ``KDTree``
-  in a backwards-compatible way.
 - ``distance_wrap.c`` needs to be cleaned up (maybe rewrite in Cython).
```

Comment:
I wonder if it is worth a note about the work by @peterbell10 et al. to rewrite the metrics in C++, since that has been coming along nicely I think. A few more metrics to be migrated over maybe?